polkadot-protocol/smart-contract-basics/.pages

```diff
@@ -0,0 +1,7 @@
+title: Smart Contract Basics
+nav:
+  - index.md
+  - overview.md
+  - polkavm-design.md
+  - evm-vs-polkavm.md
```

Comment:
Just raising this here but these should all have the names on the left. I think this is fixed in another PR